lib/debug/commands/gpio_commands.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
```

Comment:
whitespace below pragma, this might be the same as the whitespace at the end of the files in the sense we can't enforce this, depends what we want to do

Reply:
@TomLonergan03 If this is something you care about making a Github Action that greps for it is probably doable like some of the ones in lint.yml that I made for `hyped::` and std prefixes for the int types.